Firestore/Source/Model/FSTFieldValue.mm

```diff
@@ -907,15 +907,20 @@ - (NSComparisonResult)compare:(FSTFieldValue *)other {
 
 @end
 
-@interface FSTDelegateValue ()
-@property(nonatomic, assign, readonly) FieldValue internalValue;
+@interface FSTDelegateValue () {
```

Comment:
You can move the ivar declaration down to the `@implementation` block to avoid this (now useless) class extension.

Reply:
Done.